### PR TITLE
New package: fail2ban-0.8.14.

### PR DESCRIPTION
**Pull request by necrophcodr**
This provides the current stable version of fail2ban, with no batteries included.

Critiques and comments, lets have 'em!

### CODE REVIEW

**Comment by chneukirchen** [comment]
Which ssh logfile will this use?

**Comment by necrophcodr** [comment]
None. That's up to the individual to configure. No jails are included in this release, as is the case with the default package, and the Ubuntu and probably Debian packages.

All that is included are actions and filters, the rest is something one can configure.

**Comment by xtraeme** [comment]
- This package is `noarch`
- This package is missing `conf_files`
- This package is missing `pycompile_dirs`

**Comment by necrophcodr** [comment]
I know about the `conf_files` part. I did not know about the rest. Thanks!

I'm just trying to get the `conf_files` part done because that is one of those that sucks, with like 50 configuration files to enter. I'm guessing directories aren't a go in that area.

Anyway, is `noarch` and `pycompile_dirs` documented someplace? I'll read up on them then.

**Comment by necrophcodr** [comment]
How about that? Please let me know if there's a better way to do conf_dirs because oh my god

**Comment by xtraeme** [comment]
There's no way to allow wildcards for `conf_files` right now.

Would be nice to support that in xbps-src tho.

File: srcpkgs/fail2ban/template [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Template file for 'fail2ban'
+pkgname=fail2ban
+version=0.8.14
+revision=1
+noarch=yes
+build_style=python-module
+hostmakedepends="pkg-config python"
+depends="python"
+short_desc="Authentication failure monitor system"
+pycompile_dirs="/usr/share/fail2ban/client /usr/share/fail2ban/server /usr/share/fail2ban/common /usr/share/fail2ban/testcases"
```
Comment (xtraeme) [inline srcpkgs/fail2ban/template]: You can just use `pycompile_dirs=/usr/share/fail2ban"`

File: srcpkgs/fail2ban/template [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+conf_files="etc/fail2ban/action.d/apf.conf etc/fail2ban/action.d/badips.conf \
```
Comment (xtraeme) [inline srcpkgs/fail2ban/template]: `conf_files` expect an absolute path, i.e `/etc/foo`.

Comment (necrophcodr) [inline srcpkgs/fail2ban/template]: It did the thing wrong. Alright, I'll have it fixed in a bit.

**Comment by Gottox** [comment]
please rebase.

**Comment by necrophcodr** [comment]
Rebase done.

**Comment by necrophcodr** [comment]
Rebase completed. Waiting for travis to light up green.

**Comment by Gottox** [comment]
Here we go!

**Pull request merged**